sos/plugins/openstack_glance.py

```diff
@@ -44,6 +46,18 @@ def setup(self):
 
         self.add_copy_spec("/etc/glance/")
 
+        run_cmds = False
+        for os_var in ['OS_USERNAME', 'OS_PASSWORD', 'OS_TENANT_NAME']:
+            if os_var not in os.environ:
+                self.soslog.warning("%s not found in environment variables"
+                                    " which is required to run commands"
+                                    % (os_var))
+            else:
+                run_cmds = True
```

Comment:
Oh yes, I blindly c/p it from the existed nova plugin. I'll fix this and submit an updated patch. Also I will check the other openstack plugins we already have. I know e.g. the same is in gnocchi.